### PR TITLE
Removed setter

### PR DESCRIPTION
**Pull request by praneethpr**

### CODE REVIEW

**Review by fightingtexasaggie — COMMENTED** [review]
I guess this command will need to store the "new" stubfile in the new manner.

**Comment by praneethpr** [comment]
> I guess this command will need to store the "new" stubfile in the new manner.

Yes, do we want to put a ticket for it so that we don't forget about it

File: src/Controller/UI/DatasetReviewController.php [hunk 1]
```diff
@@ -598,7 +598,6 @@ protected function newDatasetFile(DatasetSubmission $datasetSubmission)
     {
         $datasetSubmission->setDatasetFileTransferStatus(DatasetSubmission::TRANSFER_STATUS_NONE);
         $datasetSubmission->setDatasetFileName(null);
```
Comment (mickel1138) [inline src/Controller/UI/DatasetReviewController.php:600]: We can probably get rid of these too soon.

**Comment by mickel1138** [comment]
> > I guess this command will need to store the "new" stubfile in the new manner.
> 
> Yes, do we want to put a ticket for it so that we don't forget about it

Yes, let's put in a ticket to deal with Cold Storage.

**Review by mickel1138 — APPROVED** [review]
Approved.

File: src/Controller/UI/DatasetSubmissionController.php [hunk 2]
```diff
@@ -318,7 +318,6 @@ protected function newDatasetFile(DatasetSubmission $datasetSubmission)
     {
         $datasetSubmission->setDatasetFileTransferStatus(DatasetSubmission::TRANSFER_STATUS_NONE);
```
Comment (mickel1138) [inline src/Controller/UI/DatasetSubmissionController.php:319]: This is also broken, this function is gone now?

**Review by mickel1138 — CHANGES_REQUESTED** [review]
Set tranfer type breaks on submission.

File: src/Controller/UI/DatasetSubmissionController.php (hunk 2 above)
Comment (praneethpr) [reply src/Controller/UI/DatasetSubmissionController.php:319]: No, we din't merge that branch. let me test again

**Comment by praneethpr** [comment]
> Set tranfer type breaks on submission.

I'm unable to reproduce the error which you are talking about, the regular dataset workflow works fine. Dataset submission works for old ones (i.e Request revisions) also

**Review by mickel1138 — APPROVED** [review]
Looks good, did a fill rebuild and cleared old files. Works now.

**Pull request merged**